### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by raihan2bd**
# Setup project 🚀 

**I made a couple of changes in this PR and some of them are:**
- [ ] At first, I created a repo and flowed a new Gitflow. ✔️ 
- [ ] Then I setup Webpack and Linters in this project ✔️ 
- [ ] Then I added the HTML structure. ✔️ 
- [ ] Then I added some Javascript functionality to display the score list. ✔️ 
- [ ] Added the professional readme file and added build files. ✔️ 
- [ ] Last but not least I deploy the page. ✔️ 

## Demo picture
project_demo

## [Live Preview](https://raihan2bd.github.io/Leaderboard/)

### CODE REVIEW

File: src/modules/manage-leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    data.forEach((item) => {
+      domUI += `<li class='score-item' id='${item.id}'>
+      ${item.name}: <span class="badge-score">${item.score}</span>
+      </li>`;
+    });
+  } else {
+    domUI = '<p class=\'no-item\'>There is not score to display</p>';
+  }
+  domContainer.innerHTML = domUI;
+};
```
Comment (DammyShittu) [inline src/modules/manage-leaderboard.js:15]: - [ ] While styling is not required, you are still expected to maintain the layout from the wireframe. Kindly fix your app layout.

  Your layout | Wireframe layout
  :---:|:---:
  dhhd | yydd

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @raihan2bd,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/manage-leaderboard.js (hunk 1 above)
Comment (raihan2bd) [reply src/modules/manage-leaderboard.js:15]: Hi! @DammyShittu Thanks a lot for your suggestion 🙏.  I will fix the wireframe next time 👍

**Review by omarsalem7 — APPROVED** [review]
Hi @raihan2bd 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ The design match wireframe guidelines
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**